dashboard/app/helpers/aichat_sagemaker_helper.rb

```diff
@@ -63,8 +63,10 @@ def self.get_sagemaker_assistant_response(aichat_model_customizations, stored_me
     inputs = format_inputs_for_sagemaker_request(aichat_model_customizations, stored_messages, new_message, level_id)
     selected_model_id = aichat_model_customizations[:selectedModelId]
     sagemaker_response = request_sagemaker_chat_completion(inputs, selected_model_id)
+    puts "sagemaker_response #{sagemaker_response}"
     parsed_response = JSON.parse(sagemaker_response.body.string)
     generated_text = parsed_response[0]["generated_text"]
+    puts "generated_text #{generated_text}"
```

Comment:
Were these left in deliberately?

Reply:
oh man - must've been on this branch while testing a different issue. Will remove in a current PR! Thanks for pointing out.